### PR TITLE
Fixes #8192: Better document Node Api: include detail always returns \"default\"

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/8192

### CODE REVIEW

File: src/api_v2/node.md [hunk 1]
```diff
@@ -102,9 +102,9 @@ HTTP/1.1 200 OK
 
     @apiParam (Mono) {Query}      query A query json object like we use as query in NodeGroups. A three field json object { 'composition' : 'and', 'select' : 'node', 'where' : [{"objectType":"node","attribute":"OS","comparator":"eq","value":"Linux"}]}, That parameter can be replace by the three following (select, composition, where) 
     @apiParam (Mono) {String}          [composition=and] Boolean operator to use between each criteria. Other value is "And"
-    @apiParam (Mono) {String}          [select=node] What kid of data we wnt to include. Here we can get policy servers/relay by setting 'nodeAndPolicyServer'
+    @apiParam (Mono) {String}          [select=node] What kid of data we want to include. Here we can get policy servers/relay by setting 'nodeAndPolicyServer'
```
Comment (jooooooon) [inline src/api_v2/node.md]: s/kid/kinf/

Comment (jooooooon) [inline src/api_v2/node.md]: Er, kind.

File: src/api_v2/node.md [hunk 2]
```diff
@@ -102,9 +102,9 @@ HTTP/1.1 200 OK
...
     @apiParam (Mono) {Criterion} where The criterion you want to find for your nodes like '[{"objectType":"node","attribute":"OS","comparator":"eq","value":"Linux"}]}'
-    @apiParam (Multi) {String}         [include=default] Level of information to include from the node inventory. Some base levels are defined (minimal, default, full). you can add fields you want by adding them to the list, possible values are keys from json answer. 
+    @apiParam (Multi) {String}         [include=default] Level of information to include from the node inventory. Some base levels are defined (minimal, default, full). you can add fields you want to a base level by adding them to the list, possible values are keys from json answer. If you don't provide a base level, they will be added to default base level, so if you only want os details, put "minimal,os" as include value
```
Comment (jooooooon) [inline src/api_v2/node.md]: "you can add fields" should have a capital ("You can...")

Comment (jooooooon) [inline src/api_v2/node.md]: s/put "minimal,os" as include value/use "minimal,os" as the value for this parameter/

**Comment by jooooooon** [comment]
Aside from these comments this looks good!

**Comment by VinceMacBuche** [comment]
PR updated

**Comment by VinceMacBuche** [comment]
All right, thanks @jooooooon ! Merging this!

**Pull request merged**